pkg/inventory/policy.go

```diff
@@ -141,11 +141,11 @@ func CanPrune(inv Info, obj *unstructured.Unstructured, policy Policy) (bool, er
 	}
 }
 
-func AddInventoryIDAnnotation(obj *unstructured.Unstructured, inv Info) {
+func AddInventoryIDAnnotation(obj *unstructured.Unstructured, inventoryID ID) {
 	annotations := obj.GetAnnotations()
 	if annotations == nil {
 		annotations = make(map[string]string)
 	}
-	annotations[OwningInventoryKey] = inv.ID()
+	annotations[OwningInventoryKey] = string(inventoryID)
```

Comment:
might be good to add a String() method to the ID, to avoid casting every time we use it, and to satisfy the Stringer interface for printing.

Reply:
Done